aws/resource_aws_route.go

```diff
@@ -350,7 +350,7 @@ func resourceAwsRouteUpdate(d *schema.ResourceData, meta interface{}) error {
 		"vpc_peering_connection_id",
 	}
 	replaceOpts := &ec2.ReplaceRouteInput{}
-
+	_ = replaceOpts
```

Comment:
@vnguyen7 instead of assigning this for no reason, we should instead convert the `replaceOpts` declaration to a variable declaration:

```
var replaceOpts *ec2.ReplaceRouteInput
```